test/JDBC/expected/BABEL-1975.out

```diff
@@ -59,9 +59,19 @@ int
 SET ROWCOUNT 1;
 SELECT @@rowcount;
 GO
-~~ERROR (Code: 33557097)~~
+~~START~~
+int
+0
+~~END~~
+
 
-~~ERROR (Message: Settings other than 0 are not allowed for option ROWCOUNT. please use babelfishpg_tsql.escape_hatch_session_settings to ignore)~~
```

Comment:
We should keep this as a previous behavior test (similar to AVG-Aggregate-Dep-before-15-2-or-14-7 in https://github.com/babelfish-for-postgresql/babelfish_extensions/commit/05b9fca3118a8f2decbebad32e48d9481ed560e3)